proseco/acq.py

```diff
@@ -9,7 +9,6 @@
 import numpy as np
 from scipy import ndimage, stats
 from scipy.interpolate import interp1d
-from astropy.table import Table
```

Comment:
I am assuming that this was imported and not used.

Reply:
Correct.

Reply:
I did run tests, but it isn't a bad idea as reviewer to just independently check out the repo in this branch and run `python setup.py test` or `pytest proseco`.